### ISSUE TITLE
A Fatal IndexError

### ISSUE THREAD

**Issue by ThywQuake** [Retinal-Research/EVIT-UNET#9]
In the file `Eff_Unet.py`, when the `fork_feat` is set `False` in `Eff_Unet`, 
you will find that in the `forward_tokens` method, the `out` is empty,
so in the `forward_feature` method, the `x_downsample` is also empty,
correspondingly in the `forward` method the variable `x_downsample`.

However, in the next method in `forward` -- `forward_up_features`, 
you reverse `x_downsample` to `x_downsample_reverse`, and then try to get item in it.
But we know it's empty since the `fork_feat` is `False`, so an `IndexError` emerges.

In my opinion, maybe the parameter `fork_feat` fixed at `True` is enough to solve the problem.

**Comment by xenosxiny** [comment]
Yes. You're right! The fork_feat should keep True. The code has been updated. I have removed this flag. Thanks!
